include/useful_qualifiers.h

```diff
@@ -0,0 +1,17 @@
+#ifndef USEFUL_QUALIFIERS__
```

Comment:
Are these used by maxmod? Might want to create an `mm_internal.h` file instead of this, as well as remove definitions not necessary for maxmod (like `MAX_OPTIMIZE`, or `IWRAM_CODE`).

Also, you should mark the file as being appropriately licensed.

Reply:
IWRAM_CODE can be useful, as it is now, to have code which is in IWRAM for the GBA, and normal for the DS.

Reply:
That's valid, though you might want to cover that in the appropriate header file if you're introducing it, then.